qm.te

```diff
@@ -1,3 +1,29 @@
 policy_module(qm, 0.6.4)
 
+gen_require(`
+	attribute container_file_type;
+	attribute container_runtime_type;
+')
+
+type ipc_t;
+domain_type(ipc_t)
+role system_r types ipc_t;
+unconfined_domain_noaudit(ipc_t)
+
+type ipc_exec_t;
+application_executable_file(ipc_exec_t)
+allow ipc_t { ipc_exec_t container_file_type}:file entrypoint;
+
+init_system_domain(ipc_t, ipc_exec_t)
+role system_r types ipc_t;
+domtrans_pattern(container_runtime_domain, ipc_exec_t, ipc_t)
```

Comment:
@rhatdan as per this logs of rpm build 
https://download.copr.fedorainfracloud.org/results/packit/containers-qm-454/fedora-40-x86_64/07612266-qm/builder-live.log.gz

You missed the `;`